test/e2e/storage/testsuites/base.go

```diff
@@ -178,7 +183,7 @@ type genericVolumeTestResource struct {
 
 var _ TestResource = &genericVolumeTestResource{}
 
-func createGenericVolumeTestResource(driver TestDriver, config *PerTestConfig, pattern testpatterns.TestPattern) *genericVolumeTestResource {
+func createGenericVolumeTestResource(driver TestDriver, config *PerTestConfig, pattern testpatterns.TestPattern, suppSizeRange volume.SizeRange) *genericVolumeTestResource {
```

Comment:
We need to update test/e2e/storage/testsuites/disruptive.go also which calls this function for passing pull-kubernetes-typecheck job like `ERROR(all) test/e2e/storage/testsuites/disruptive.go:89:73: too few arguments in call to createGenericVolumeTestResource`

Reply:
🍺I used GoLand as my IDE. Seems it did't  highlight the error. Anyway, I think they should have been fixed now.